modules/core/arrow-mtl/src/main/kotlin/arrow/mtl/typeclasses/TraverseFilter.kt

```diff
@@ -0,0 +1,20 @@
+package arrow.mtl.typeclasses
+
+import arrow.Kind
+import arrow.core.*
+import arrow.typeclasses.Applicative
+import arrow.typeclasses.Traverse
+
+interface TraverseFilter<F> : Traverse<F>, FunctorFilter<F> {
```

Comment:
These classes have not been detected by git as a move but instead as a copy pasta and that is probably removing the history. Any way we can preserve the history?